src/components/common/Table/index.tsx

```diff
@@ -58,82 +64,99 @@ export default function Table<T>(props: Props<T>) {
       wrapper={(children) => (
         <Card
           className={classNames('w-full', props.type !== 'balances' && 'h-fit')}
-          contentClassName='max-w-full overflow-x-scroll scrollbar-hide'
+          contentClassName='max-w-full overflow-x-auto scrollbar-hide'
           title={props.title}
         >
           {children}
         </Card>
       )}
     >
-      <table className={classNames('w-full', props?.tableBodyClassName)}>
-        <thead className='border-b bg-black/20 border-white/10'>
-          {table.getHeaderGroups().map((headerGroup) => (
-            <tr key={headerGroup.id}>
-              {headerGroup.headers.map((header) => {
-                return (
-                  <th
-                    key={header.id}
-                    onClick={header.column.getToggleSortingHandler()}
-                    className={classNames(
-                      props.spacingClassName ?? 'px-4 py-3',
-                      header.column.getCanSort() && 'hover:cursor-pointer',
-                      LEFT_ALIGNED_ROWS.includes(header.id) ? 'text-left' : 'text-right',
-                      header.column.columnDef.meta?.className,
-                    )}
-                  >
-                    <div
-                      className={classNames(
-                        'flex',
-                        LEFT_ALIGNED_ROWS.includes(header.id) ? 'justify-start' : 'justify-end',
-                        'align-center relative',
-                      )}
-                    >
-                      <Text
-                        tag='span'
-                        size='xs'
-                        className='flex items-center font-normal text-white/60'
-                      >
-                        {flexRender(header.column.columnDef.header, header.getContext())}
-                      </Text>
-                      {header.column.getCanSort() && (
-                        <span
+      <div className='w-full'>
+        <div className='w-full overflow-x-auto'>
```

Comment:
Prevent horizontal scrollbar
```suggestion
        <div className='w-full overflow-x-auto scrollbar-hide'>
```